app/src/main/res/values/ic_app_name_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Qué es esto? No veo que se use en ningún lado.

Reply:
Aaaaaa me quedo este archivo entre las pruebas que hice para que funque, ahi lo borro!
Esto se creaba cuando importaba el texto de Play to Play del splash como un image asset